starter-code/lib/app.js

```diff
@@ -27,6 +27,23 @@ const app = net.createServer(client => {
         body:'<html><body><h1>This is blue</h1></body></html>'
       }));
 
+    } 
+
+    else if(request.path === '/echo') {
+      client.end(createResponse({
+        contentType:'text/html',
+        status:'200 OK',
+        body:'<html><body><h1>This is blue</h1></body></html>'
+      }));
+
```

Comment:
For you echo else if statement, i would change the body to something other than "this is blue", maybe "this is echo" or something like that, because at first test through postman i thought it was broken.